### PR TITLE
Enforce strong passwords coming from requests

### PR DESCRIPTION
**Pull request by johnpcooke94**
This PR will enforce strong passwords on the back-end. This was tested with a small unit test written to check password enforcement, as well as altering request data and ensuring that the proper response was received from the back-end.

Password strength enforcement was achieved using [this](https://www.nuget.org/packages/zxcvbn-core/) library, which is a port to .NET of the same library that is used for enforcement on the front-end. So if there is a password that fails the strength enforcement on the front-end, it should also fail the enforcement on the back-end, should the request still get through somehow.

Feel free to suggest test cases for the unit test, and I will gladly add them.

### CODE REVIEW

File: CSLabs.Api/RequestModels/RegistrationRequest.cs [hunk 1]
```diff
@@ -1,10 +1,9 @@
-using System;
-using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
+using System.Reflection;
 using CSLabs.Api.Models;
 using CSLabs.Api.Util;
-using Microsoft.EntityFrameworkCore;
```
Comment (Cliftonz) [inline CSLabs.Api/RequestModels/RegistrationRequest.cs:7]: Why was this removed?
I assume you are not calling any db objects?

**Review by Cliftonz — APPROVED** [review]
Love the tests you added!!
Lgtm, just respond to the comment.

File: CSLabs.Api/RequestModels/RegistrationRequest.cs (hunk 1 above)
Comment (johnpcooke94) [reply CSLabs.Api/RequestModels/RegistrationRequest.cs:7]: Yeah, everything I removed here was highlighted as unused by the linter. I'm not sure why this was included, presumably it was used at one point and then whatever was calling it got removed. So since I was working in the file anyway I just went through and removed all the unused imports.

There is a method that uses a db context, but that's passed in as an argument and so will need to be qualified by the caller, but there's no need to import it here.

Comment (Cliftonz) [reply CSLabs.Api/RequestModels/RegistrationRequest.cs:7]: Sounds good, thank you for doing that.

**Review by jasekiw — APPROVED** [review]
